File: app/src/main/java/com/neome/api/ent/entAside/msg/MsgEntUserPickerCandidateListGet.java
// neome.ai API - do not change
//

package com.neome.api.ent.entAside.msg;

import org.jetbrains.annotations.Nullable;

import com.neome.api.meta.base.dto.FormValueRaw;
import com.neome.api.meta.base.Types.MetaIdForm;
import com.neome.api.meta.base.Types.MetaIdRole;
import com.neome.api.meta.base.Types.MetaIdVar;
import com.neome.api.nucleus.base.msg.Msg;

import java.util.Set;

@SuppressWarnings("unused")
public class MsgEntUserPickerCandidateListGet extends Msg
{
  @Nullable
  public FormValueRaw formValueRaw;

  @Nullable
  public Set<MetaIdRole> roleIdSet;

  @Nullable
  public MetaIdVar setOfUserVarId;

  public MetaIdForm sourceFormId;
}

File: app/src/main/java/com/neome/api/ent/entAside/msg/MsgEntUserPickerCandidateListGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entAside.msg

import com.neome.api.meta.base.dto.FormValueRaw
import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdRole
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.nucleus.base.msg.Msg

interface MsgEntUserPickerCandidateListGet : Msg
{
  val formValueRaw: FormValueRaw?
  val roleIdSet: Set<MetaIdRole>?
  val setOfUserVarId: MetaIdVar?
  val sourceFormId: MetaIdForm
}